students/12th/eunsulim/style/login.css

```
@@ -0,0 +1,268 @@
+* { 
+    list-style-type: none;
+    text-decoration: none;
+    padding: 0;
+    margin: 0;
+    box-sizing:border-box;
+}
+
+html, body {
+    height:100%;
+    background-color: #fafafa;
+}
+
+.wrap {
+    display: flex;
+    justify-content: center;
+    height: 100%;
+}
+
+.container {
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+}
+
+.content {
+    display: flex;
+    flex-grow: 1;
+}
+
+.innerBox {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 30px auto 0 ;
+    padding-bottom: 44px;
```

```suggestion
    margin: 30px auto 44px ;
```
이 경우에는 padding이나 margin 둘 중에 하나만 사용하시는 거 더 깔끔할 거 같네요!

---

넵 margin 만 사용해서 변경했습니다.